### ISSUE TITLE
Genes in enriched pathways

### ISSUE THREAD

**Issue by ziyan109** [arc85/singleseqgset#2]
Hello,

This is such a great tool and I was able to generate the heatmap :)
I'm wondering is there any function to pull out the genes that are enriched in certain pathway? For example, TGF-beta signaling pathway is enriched in one of my clusters, I wonder which genes are up-regulated in there.

Thanks so much.

**Comment by arc85** [comment]
Hi Ziyan, great point. It would be helpful to explore "leading edge" genes directly from this package.

**Comment by abedkurdi** [comment]
Excuse me, but where to find the function related to "leading edge"?

**Comment by ziyan109** [comment]
Hi,
I was referring to the "leading edge gene" returned by the fgsea package.
Is it possible to do it in singleseqgset?

Thanks!

**Comment by abedkurdi** [comment]
Yeah, I guess, but no idea how. I will give it a try when I have some time.
